tests/accelerator/sycl/common/test_sycl_basics.hpp

```diff
@@ -165,5 +165,6 @@ TYPED_TEST_P(test_sycl_basics, transform3) {
   this->compareOutputs();
 }
 
-REGISTER_TYPED_TEST_SUITE_P(test_sycl_basics, vector_2d_ops, vector_3d_ops,
-                            matrix64_ops, matrix22_ops, transform3);
+REGISTER_TYPED_TEST_SUITE_P(test_sycl_basics, /*vector_2d_ops, vector_3d_ops,
+                            matrix64_ops, matrix22_ops,*/
+                            transform3);
```

Comment:
How about, instead of turning off the build, turning off the running of the tests instead? 🤔 I would still like to keep the build tests alive...

Reply:
Done